e2e/protect-app/src/protect-marketplace.ts

```diff
@@ -0,0 +1,212 @@
+/*
+ *
+ * Copyright © 2025 Ping Identity Corporation. All right reserved.
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ *
+ */
+
+import './style.css';
+import { protect } from '@pingidentity/protect';
+import {
+  CallbackType,
+  Config,
+  FRAuth,
+  FRStep,
+  FRUser,
+  NameCallback,
+  PasswordCallback,
+  TokenManager,
+  UserManager,
+} from '@forgerock/javascript-sdk';
+
+const protectAPI = await protect({ envId: '02fb4743-189a-4bc7-9d6c-a919edfe6447' });
+const FATAL = 'Fatal';
+
+await Config.setAsync({
```

Comment:
So typically for e2e apps, we try to make them general to the config.

I'm not opposed to this because we need this tenant for now, but just calling out that we usually don't hardcode the config directly into the e2e so we can swap it in and out with an `.env` file